config/config.yml

```diff
@@ -131,6 +131,7 @@ config:
             signing_key: "base64 encoded HMAC key or public RSA/ECDSA pem key"
             jwt_header: "Authorization"
             jwt_url_parameter: null
+            clockSkewToleranceSeconds: 300
```

Comment:
Nit: What do you think about using underscores to match the rest of the parameters in this section?

Reply:
ok

Reply:
Why 300s here? Shouldn't a lower number, maybe 60s, be sufficient?

@cwperks @peternied Thoughts on this?

Reply:
I searched online for which default values are commonly used , e.g.
https://learn.microsoft.com/en-us/dotnet/api/microsoft.identitymodel.tokens.tokenvalidationparameters.defaultclockskew?view=msal-web-dotnet-latest